index.html

```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <h1>Riley Fan Page</h1>
+
+    <p>How is your guinea pig doing?</p>
```

Comment:
She's doing well! Thanks for asking. She's a grumpy old lady and keeps fighting with the other pigs haha.